packages/opentelemetry/package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "@itwin/presentation-opentelemetry",
+  "version": "4.0.0-dev.76",
+  "description": "Utilities to enable OpenTelemetry tracing of Presentation requests",
+  "license": "MIT",
+  "engines": {
+    "node": "^18.0.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/iTwin/itwinjs-core.git",
```

Comment:
Need to update these to point to the new repo

Reply:
fixed